python-backend/app/api/mines/mine/models/mine.py

```diff
@@ -40,6 +40,11 @@ class Mine(AuditMixin, Base):
         lazy='joined')
     mine_type = db.relationship(
         'MineType', backref='mine', order_by='desc(MineType.update_timestamp)', lazy='joined')
+    mine_party_appt = db.relationship(
```

Comment:
I really don't like seeing MinePartyAppointment on the mine model, which is already extremely heavy. In various API discussions we talked about treating each resource (/mines, /parties, /documents) as if it were a separate micro-service. Either loop back and call the /parties endpoint to fetch this data if it's really needed here (there is a pattern to copy in our API for this), or have the front-end fetch it from /parties when it's needed. It's already in the store and there are actions around it.

Reply:
Actually it doesn't look like you're returning party appointments with the mine. I'm curious why you needed the relationship defined here?

Reply:
This was a requirement when I was leveraging the relationship in the `MinePartyAppointment` model to query for the related mine, prior to switching to the internal API request. I don't think this is needed anymore and should be removed. Good catch